group_vars/rhcs.yml.sample

```diff
@@ -114,6 +114,10 @@ fetch_directory: ~/ceph-ansible-keys
 #ceph_stable_release: dummy
 #ceph_stable_repo: "{{ ceph_mirror }}/debian-{{ ceph_stable_release }}"
 
+#nfs_ganesha_stable: true # use stable repos for nfs-ganesha
```

Comment:
@leseb do the vars added in this file also need to be copied over to group_vars/all.yml.sample?